pkg/kyverno/apply/command.go

```diff
@@ -73,7 +73,7 @@ func Command() *cobra.Command {
 					os.Exit(3)
 				}
 				if policyHasVariables(*policy) {
-					return sanitizedError.New(fmt.Sprintf("Policy %v is not valid - 'apply' does not support policies with variables", policy.Name))
+					return sanitizedError.NewWithError(fmt.Sprintf("Policy %v is not valid - 'apply' does not support policies with variables", policy.Name), err)
```

Comment:
change to, "invalid policy %s. "apply" does not support policies with variables

Reply:
updated accordingly